cmd/backup-manager/app/backup/manager.go

```diff
@@ -323,9 +323,19 @@ func (bm *Manager) performBackup(ctx context.Context, backup *v1alpha1.Backup, d
 	case string(v1alpha1.BackupModeVolumeSnapshot):
 		// In volume snapshot mode, commitTS and size have been updated according to the
```

Comment:
we should remove the content about size in comment.

Reply:
fixed in https://github.com/pingcap/tidb-operator/pull/4819/commits/fc18e21a421dee08cf3257099b503047e5d552a7